src/backend/assets/js/social.js

```diff
@@ -0,0 +1,17 @@
+$('.social-button').click(function() {
+  var source = this.id.toString().split('-')[0];
+
+  switch (source) {
+    case 'fb':
+      window.open('http://www.facebook.com/share.php?u=' + window.location, 'facebook', 'height=360, width=580');
+      break;
+    case 'tw':
+      window.open('http://twitter.com/home?status=' + window.location, 'twitter', 'height=360, width=580');
+      break;
+    case 'go':
+      window.open('https://plus.google.com/share?url=' + encodeURIComponent(window.location), 'google', 'height=360, width=580');
```

Comment:
Google plus was not working without encoding the url. So I added the it to encode the url.